head_test.go

```diff
@@ -29,7 +30,8 @@ import (
 )
 
 func BenchmarkCreateSeries(b *testing.B) {
-	lbls, err := labels.ReadLabels("testdata/20kseries.json", b.N)
+	seriesPath := filepath.Join("testdata", "20kseries.json")
+	lbls, err := labels.ReadLabels(seriesPath, b.N)
```

Comment:
no need for a separate var
`lbls, err := labels.ReadLabels(filepath.Join("testdata", "20kseries.json"), b.N)`

the same for the other places

Reply:
Thanks for the feedback, @krasi-georgiev ! I used a regular expression to try to find those path string literal occurrences in the code base, but I'll double-check later this evening and will confirm with you that we got them all. Also, I will make the change you suggested and remove the separate vars.

Thanks & regards,
- Ben

Reply:
I'm glad you had me double-check the code. It turns out there were 3 other lines in `wal_test.go` that also needed to be updated that I missed in the first commit. I also went ahead and inlined those separate vars as per your suggestion.

Thanks again for the guidance.